src/components/common/calendar/Calendar.js

```diff
@@ -27,15 +26,11 @@ function CalendarApp() {
   const [eventFlag, setEventFlag] = useState(true);
   const [form] = Form.useForm();
 
-  const { authState, oktaAuth } = useOktaAuth();
-  const { idToken } = authState;
-
-  // const token = oktaAuth.getIdToken();
+  // const { idToken } = authState;
 
+  // TO-DO: Implement Auth0
   useEffect(() => {
     if (eventFlag) {
-      console.log(authState);
-      console.log(oktaAuth);
       axiosWithAuth(idToken)
```

Comment:
Looks like we're throwing an error here because `idToken is not defined` - let's touch this file up to make sure this code runs as it should!